pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -685,6 +685,66 @@ func TestPlugin(t *testing.T) {
 			},
 		},
 
+		"missing-translated-class-parameters": {
+			pod:     podWithClaimName,
+			claims:  []*resourcev1alpha2.ResourceClaim{claimWithCRD(pendingDelayedClaimWithParams)},
+			classes: []*resourcev1alpha2.ResourceClass{classWithCRD(structuredResourceClassWithCRD)},
+			objs:    []apiruntime.Object{claimParameters /* classParameters, */, workerNodeSlice},
```

Comment:
```suggestion
			objs:    []apiruntime.Object{claimParameters, workerNodeSlice},
```

Reply:
This is done intentionally, it makes the code more readable. cc @pohly do we need to drop it?

Reply:
Let's remove it. There's no need to document that the test case has no class parameters.